packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -47,6 +47,11 @@ export interface DataListProps extends Omit<React.HTMLProps<HTMLUListElement>, '
   wrapModifier?: DataListWrapModifier | 'nowrap' | 'truncate' | 'breakWord';
   /** @deprecated Order of items in a draggable DataList */
   itemOrder?: string[];
+  /** @beta Object that causes the data list to render hidden inputs which improve selectable item a11y */
+  selectableInput?: {
```

Comment:
I like this API better!

Reply:
As do I, it was a great suggestion!

Reply:
I like Coker's suggestion.  Also, can we create an interface for the object so we can document each property in the object like we do in [slider](https://github.com/patternfly/patternfly-react/blob/c2faf7db0e1006d1660d572d7f507053685cb9b8/packages/react-core/src/components/Slider/Slider.tsx#L10).  `selectabelInput` would thenhave the type of whatever you call the interface. You can add the interface to the `propComponents` in the md file so it is documented on the website.